packages/react-lowcode/src/codegen/generation/interfaces/naming.ts

```diff
@@ -0,0 +1,31 @@
+import { camalizeString } from "../../../strings/camel";
+
+export interface Naming {
```

Comment:
what about just having two interfaces `ComponentNaming` and `EntityNameMapper`

what about `entityFolderName(entityName)`

```
interface CrudComponents {
  list(entityName): { 
    // reusable view only component     
    page: { componentName, componentPath}, 
    // page/routing components (composite components)
    view: { componentName, componentPath} 
  }
  detail(entityName): { page: { componentName, componentPath}, view: { componentName, componentPath} }
}

// EntityNameMapper implementation would use:
interface FileNaming {
    // upperCamelCase or lower case with dashes...
   componentFileName(componentName): string
   readonly srcFolder: string // returns // src
   readonly pagesPath: string // returns // src/pages, src/routing
   // more will come soon (maybe shared components...)
}
interface ComponentNaming {
  formatComponentName(entityName, stereotype): string
}
```

intentionally:
- without get* prefixes
- split to multiple interfaces